.FDm/config.local.toml

```diff
@@ -12,6 +12,7 @@ player_file_name = "players.csv"
 repo_path="resources/prod/app_config/rewards_configuration"
 generated_path="data/generated/prod/rewards"
 file_name = "rewards.csv"
+focus_players_file = "focus_players.csv"
```

Comment:
File name for uploading a list of focus players for the app - for now, this is one static list of player keys.